src/components/Drawer.js

```diff
@@ -152,11 +159,11 @@ const Drawer = React.createClass({
         backgroundColor: StyleConstants.Colors.PORCELAIN,
         boxShadow: StyleConstants.ShadowHigh,
 
-        '@media (max-width: 750px)': {
+        [`@media (max-width: ${this.props.breakPoints.medium}px)`]: {
```

Comment:
He is doing string interpolation here so he has to surround the string in brackets in order for the style object to behave correctly.  At least that is how I interpret this.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals